assets/js/components/Dashboard/PaMessagesPage.tsx

```diff
@@ -147,55 +147,42 @@ const PaMessagesPage: ComponentType = () => {
                 <BoxArrowUpRight />
               </Link>
             </section>
-            <section>
-              <header>Filter by message state</header>
-              <ButtonGroup className="button-group" vertical>
-                <Button
-                  className={cx("button", {
-                    active: stateFilter === "current",
-                  })}
-                  onClick={() => setStateFilter("current")}
-                >
-                  Now
-                </Button>
-                <Button
-                  className={cx("button", { active: stateFilter === "future" })}
-                  onClick={() => setStateFilter("future")}
-                >
-                  Future
-                </Button>
-                <Button
-                  className={cx("button", { active: stateFilter === "past" })}
-                  onClick={() => setStateFilter("past")}
-                >
-                  Done
-                </Button>
-              </ButtonGroup>
-            </section>
-            <section>
-              <header>Filter by service type</header>
-              <ButtonGroup className="button-group" vertical>
-                <Button
-                  className={cx("button", {
-                    active: serviceTypes.length === 0,
-                  })}
-                  onClick={() => setServiceTypes([])}
-                >
-                  All
-                </Button>
-                {SERVICE_TYPES.map((serviceType) => (
-                  <Button
-                    key={serviceType}
-                    className={cx("button", {
-                      active: serviceTypes.includes(serviceType),
-                    })}
-                    onClick={() => setServiceTypes([serviceType])}
-                  >
-                    {getServiceLabel(serviceType)}
-                  </Button>
-                ))}
-              </ButtonGroup>
-            </section>
+            <FilterGroup
+              header="Filter by message state"
+              onFilterSelect={(selected) => {
+                const stateFilter = selected.toLowerCase() as StateFilter;
```

Comment:
These values are already lowercase, so this `toLowerCase` and the one below seem unnecessary.